tasks/hidden_service.yml

```diff
@@ -4,7 +4,7 @@
 
 - name: Add the HiddenServiceDir config to torrc
   blockinfile:
-    dest: /etc/tor/torrc
+    dest: "{{ torrc }}"
```

Comment:
Where is this variable defined? I don't see vars/ (maybe you forgot to git add them)